decimalSuite_test.go

```diff
@@ -173,27 +221,27 @@ func runTest(testVals decValContainer, testValStrings testCaseStrings) (Decimal6
 
 // TODO: get runTest to run more functions such as FMA.
 // execOp returns the calculated answer to the operation as Decimal64.
-func execOp(val1, val2, val3 Decimal64, op string) Decimal64 {
+func execOp(context Context64, testVals decValContainer, op string) Decimal64 {
```

Comment:
Still struggling with the change from three vars to `testsVals`. The nett effect is more code and harder to read. I would even suggest going the other way and calling the parameters `a`, `b` and `c`.